apps/sms/views/inbox/js/inbox.js

```diff
@@ -398,8 +398,9 @@ var InboxView = {
     selected.forEach((id) => {
       var thread = Threads.get(+id);
 
-      var markable = thread && !thread.isDraft &&
-        (isRead || !thread.getDraft());
+      var markable = thread && !thread.isDraft && 
+              (isRead || !thread.getDraft()) && 
```

Comment:
Identation issue. Please follow the identation. if you follow it then it won't come in third line.
<pre>
var markable = thread && !thread.isDraft && 
   (isRead || !thread.getDraft()) && !(thread.unreadCount ^ isRead));
</pre>